### PR TITLE
Meteor pre-install

### LINKED ISSUE #8: meteor should be pre-installed before build
during build we see this output:

```
mop, this is your first time using Meteor!
Installing a Meteor distribution in your home directory.
######################################################################## 100.0%
```

Since `meteor` is actually burned into the image, need to make it so the `mop` user has access to this preinstalled version.

### PR DESCRIPTION
**Pull request by theworkflow**
- Change $HOME and give mop user owner perms

Closes #8

### CODE REVIEW

File: bin/build [hunk 1]
```diff
@@ -4,18 +4,21 @@ set -e
...
 mkdir -p $OUTPUT_DIR
 
 if [[ ! -d $INPUT_DIR ]] || [[ ! -d $OUTPUT_DIR ]]; then
-  echo "Input/Output directories must exist.\n"
+  printf "Input/Output directories must exist.\n"
   exit 1
 fi
 
+# Copy Meteor distribution to $HOME
+cp -R /opt/home /mnt
```
Comment (jackboberg) [inline bin/build]: in the `DNX` image we do: `cp -R /opt/home/. $HOME`, but you only need `.meteor`, right?

Comment (theworkflow) [inline bin/build]: That's correct. I'll update

File: bin/build [hunk 2]
```diff
@@ -24,6 +27,7 @@ source /opt/nvm/nvm.sh
 # --json || --debug
 
 cd $INPUT_DIR
+printf "Using $(meteor --version)"
```
Comment (jackboberg) [inline bin/build]: `Using Meteor 1.3Demeteorizing application...`

Need that `\n` if you want to use `printf`...

**Comment by franvarney** [comment]
👍

**Comment by tzmanics** [comment]
👍

**Comment by jackboberg** [comment]
@harlanj This approach still suffers from re-installing and using the version of Meteor in the applications `.meteor/release` file, right?

If so, do we want to check and error out _before_ attempting `--server-only`?

File: bin/build [hunk 3]
```diff
@@ -4,26 +4,31 @@ set -e
...
+# Copy Meteor distribution to $HOME
+printf "Initializing Meteor environment.\n"
+cp -R /opt/home/.meteor $HOME
+
 # Demeteorizer flags
 # TODO Test for demeteorizer flags
 # --json || --debug
 
 cd $INPUT_DIR
+printf "Using $(meteor --version).\n"
```
Comment (theworkflow) [inline bin/build:31]: This is not output on all builds. I'm testing with the Meteor Leaderboard example app with `Meteor@1.2.1`

**Comment by jackboberg** [comment]
:shipit:

**Pull request merged**